### ISSUE TITLE
Provide CIF outputs in the JSON "out" field

### ISSUE THREAD

**Issue by muhrin** [mpds-io/mpds-api#7]
Give CIF in 'out' from JSON, so we get the other information. I'm not sure about this but there is some useful information, e.g., npages, count, etc in the JSON but not if if you ask for fmt=cif. Maybe the return could be:

e.g.:

`{
    "count":44640,
    "out":[
        "<Put the CIFs here one by one>"
    ],
    "npages":4464,
    "page":0,
    "error":null
}`

**Comment by blokhin** [comment]
Then technically this is not a CIF anymore. You then need json parser and so on. I think about adding aux info to the response headers, but not sure if it is a common practice.

**Comment by muhrin** [comment]
Yes, I see what you mean, it's not ideal to force the user to have a JSON parser if they don't have one.  My concern is that without communicating the number of pages and the current page the interface is close to unusable.  I would have to keep upping the requested page until I didn't get a response anymore or something like that.

Perhaps there could be metadata in the CIF in comments, e.g.:

```
# response_count = 44640
# response_npages = 4464
# response_page = 2
...
```

**Comment by blokhin** [comment]
Absolutely true, agreed.

**Comment by blokhin** [comment]
To be released in version 0.0.5
